tests/data/preview/comments9.py

```diff
@@ -0,0 +1,240 @@
+# Test for https://github.com/psf/black/issues/246.
+
+some = statement
+# This comment should be split from the statement above by two lines.
+def function():
```

Comment:
Could we also test two lines of comments? I'd expect them both to be moved.

Reply:
Done